src/features/applicationMetadata/index.d.ts

```diff
@@ -29,7 +29,7 @@ export interface IOnEntry {
   instanceSelection?: IInstanceSelection;
 }
 
-export type ShowTypes = 'new-instance' | 'select-instance' | string;
+export type ShowTypes = 'new-instance' | 'select-instance' | 'stateless' | 'stateless-anon';
```

Comment:
I don't think this quite correct. The show-type can be any string, not just 'stateless' (which then points to the layout set of the same name) and for some strange reason the setting for allowing anonymous users is defined on the _data type_ for that layout set. See the implementation of `useAllowAnonymous()`. So the previous type was correct, even though it reduces the possibilities to just 'string'.